src/main/java/service/impl/CsvFileWriterImpl.java

```diff
@@ -0,0 +1,33 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import service.FileWriter;
+
+public class CsvFileWriterImpl implements FileWriter {
+    private static final String TITLES = "fruit,quantity";
+    private static final String PATH = "src/main/resources";
+
+    @Override
+    public void writeToFile(List<String> report, String filename) {
+        File file = new File(PATH + File.separator + filename);
+        String lines = convertToStringFormat(report);
+        try (BufferedWriter bufferedWriter
+                     = new BufferedWriter(new java.io.FileWriter(file, true))) {
```

Comment:
why true flag in constructor? Do we need to append? 🤔 
Also, consider a different name for your interface so you don't need to use fully qualified name here